eva_submission/eload_ingestion.py

```diff
@@ -66,13 +67,23 @@ def ingest(
         clustering_performed_on_assembly = None
         self.eload_cfg.set(self.config_section, 'ingestion_date', value=self.now)
         self.project_dir = self.setup_project_dir()
+
+        if not tasks:
+            tasks = copy(self.all_tasks)
+            tasks.remove('archive_only')
+
+
+        if 'archive_only' in tasks:
+            # Remove any other tasks if archive_only is present
+            tasks = ['archive_only']
+
         # Pre ingestion checks
         self.check_aggregation_done()
         self.check_brokering_done()
-        self.check_variant_db()
-
-        if not tasks:
-            tasks = self.all_tasks
+        if  'archive_only' not in tasks:
+            self.check_variant_db()
+        else:
+            self.archive_only()
```

Comment:
Not sure we need resume for a one step workflow, but if we have it we should pass it here.

Reply:
Passing it now.